src/Esfa.Vacancy.Application/Commands/CreateApprenticeship/MinimumWageCalculator.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+
+namespace Esfa.Vacancy.Application.Commands.CreateApprenticeship
+{
+    public class MinimumWageCalculator : IMinimumWageCalculator
+    {
+        private const decimal WeeksPerMonth = 4m;
+        private const decimal WeeksPerYear = 52m;
+        private const string MissingMinWageErrorMessage = "MinWage can't be null.";
```

Comment:
Are these validation messages that do not surface?

Reply:
They will be logged, but not included in the response to the client.